src/common/indexedDB/idb.ts

```diff
@@ -29,6 +29,14 @@ export class Store {
     }
 }
 
+type IDBResultTarget = EventTarget & {
+    result: IDBValidKey[];
+};
+
+type IDBEvent = {
+    target: IDBResultTarget;
+} & Event;
+
```

Comment:
Is there a reason `IDBResultTarget` takes the form `otherType & { … }` but `IDBEvent` takes the form `{ … } & otherType`? If we'd like them to be similar, do we have a preference for either order?

Reply:
There is no preference. I am not sure but I think they mean the same. So I have no preference and I have seen both used in our code.

Reply:
We should have a preference! And of course consistency in the file is the minimum bar!

Reply:
From preference I meant was a repo-wide preference and that I dont have a personal pref for this. As far as consistency in the file is concerned, I will change this instance.

Reply:
Done.